packages/nimble-components/src/rich-text/editor/styles.ts

```diff
@@ -80,6 +80,15 @@ export const styles = css`
         border: ${borderWidth} solid rgba(${borderRgbPartialColor}, 0.1);
     }
 
+    ${/* Shared styles for all mention views at edit time. */ ''}
+    .nimble-mention-view-edit {
+        color: ${bodyFontColor};
+    }
+
+    :host([disabled]) .nimble-mention-view-edit {
+        color: ${bodyDisabledFontColor};
+    }
```

Comment:
These are styles associated with the ProseMirror inserted content so for the styles to be in document order it should be aligneed with those down below:


```suggestion
    .ProseMirror .nimble-mention-view-edit {
        color: ${bodyFontColor};
    }

    :host([disabled]) .ProseMirror .nimble-mention-view-edit {
        color: ${bodyDisabledFontColor};
    }
```

Reply:
Moved the styles below to anchor styles and updated as suggested.